libqtile/layout/xmonad.py

```diff
@@ -237,16 +237,16 @@ def cmd_normalize(self, redraw=True):
             self.group.layout_all()
         self.do_normalize = False
 
-    def cmd_reset(self, redraw=True):
+    def cmd_reset(self, ratio=None, redraw=True):
```

Comment:
Can you add a `cmd_set_ratio(ratio)` method which could call `self.cmd_reset(ratio=ratio)`.

Reply:
I am not sure we should use `cmd_reset` for that, since it also normalizes secondary windows (which may be counter-intuitive). I can write a method that will just set the ratio without touching the secondaries though, should not be too hard.